ranking/src/main/java/org/apache/sdap/mudrod/ranking/traindata/ExpertRankTrainData.java

```diff
@@ -143,7 +151,7 @@ public static void calculateVec(String[][] arr) {
     {
       for (int i = 1; i < arr.length - row; i++) {
         List<String> colList = new ArrayList<String>(); // create vector to store all values inside of a column, which is stored inside 2D vector
-        for (int col = 0; col < arr[0].length - 1; col++) // Columns go until the next to last column
+        for (int col = 1; col < arr[0].length - 2; col++) // Columns go until the next to last column
```

Comment:
Why has this changed?

Reply:
Before Yongyao deleted the first column of the experts provided data manually and then invoked the function to process data.

Reply:
wow, ok thank you for fixing.